pkg/repository/olap/types.go

```diff
@@ -0,0 +1,29 @@
+package olap
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type WorkflowRun struct {
+	Id           uuid.UUID `json:"id"`
+	TaskId       int32     `json:"task_id"`
+	WorkerId     int32     `json:"worker_id"`
+	TenantId     uuid.UUID `json:"tenant_id"`
+	Status       string    `json:"status"`
+	Timestamp    time.Time `json:"timestamp"`
+	CreatedAt    time.Time `json:"created_at"`
+	RetryCount   int32     `json:"retry_count"`
+	ErrorMessage string    `json:"error_message"`
+}
+
+type Event struct {
+	TaskId     int32     `json:"task_id"`
```

Comment:
Let's store both the task's `int64` id (it's a bigint), and the task's UUID. It's going to be much harder to remove UUIDs from the API than from the engine.

Reply:
is the task's uuid the `ExternalID`?